spec/dor/publishable_spec.rb

```diff
@@ -220,6 +224,45 @@ class DescribableItem < ActiveFedora::Base
           expect(File).to_not exist(druid1.path)
         end
       end
+      
+      it 'handles externalFile references' do
+        correctPublicContentMetadata = Nokogiri::XML(read_fixture('hj097bm8879_publicObject.xml')).at_xpath('/publicObject/contentMetadata').to_xml               
+        @item.contentMetadata.content = read_fixture('hj097bm8879_contentMetadata.xml')
+        
+        # setup stubs for child items
+        %w(cg767mn6478 jw923xn5254).each do |druid|
+          child_item = ItemizableItem.new(:pid => "druid:#{druid}")
+
+          # load child content metadata fixture
+          dsid = 'contentMetadata'
+          child_item.datastreams[dsid] = Dor::ContentMetadataDS.from_xml read_fixture("#{druid}_#{dsid}.xml")
+
+          # load child DC metadata fixture and set label
+          dsid = 'DC'
+          child_item.datastreams[dsid] = ::SimpleDublinCoreDs.from_xml read_fixture("#{druid}_#{dsid}.xml")
+          child_item.label = child_item.datastreams[dsid].title
+
+          # stub out retrieval for child item
+          allow(Dor::Item).to receive(:find).with(child_item.pid).and_return(child_item)
+        end
+        
+        # generate publicObject XML and verify that the content metadata portion is correct
+        expect(Nokogiri::XML(@item.public_xml).at_xpath('/publicObject/contentMetadata').to_xml).to be_equivalent_to(correctPublicContentMetadata)
+      end
+
+      it 'handles bad externalFile references' do
+        @item.contentMetadata.content = <<-EOXML
+        <contentMetadata objectId="hj097bm8879" type="map">
+          <resource id="hj097bm8879_1" sequence="1" type="image">
+            <externalFile fileId="2542A.jp2" objectId="druid:cg767mn6478"/>
+            <relationship objectId="druid:cg767mn6478" type="alsoAvailableAs"/>
+          </resource>
+        </contentMetadata>        
+        EOXML
+        
+        # generate publicObject XML and verify that the content metadata portion is invalid
+        expect { Nokogiri::XML(@item.public_xml) }.to raise_error(ArgumentError)
+      end
```

Comment:
Where is the `ArgumentError` handled and how should it be handled?  Can this spec `expect` something to handle it?